src/commands/cycle.js

```diff
@@ -0,0 +1,67 @@
+import loadCommand from '../util/loadCommand'
+import errorReporter from '../util/errorReporter'
+import assertFunctions from '../util/assertFunctions'
+import graphQLFetcher from '../util/graphQLFetcher'
+import getServiceBaseURL, {GAME} from '../util/getServiceBaseURL'
+import defaultInvokeOptions from '../util/defaultInvokeOptions'
+
+export const {parse, usage, commandDescriptor} = loadCommand('cycle')
+
+function invokeUpdateCycleStateAPI(state, lgJWT) {
+  const mutation = {
+    query: 'mutation($state: String!) { updateCycleState(state: $state) { id } }',
+    variables: {state},
+  }
+  return graphQLFetcher(lgJWT, getServiceBaseURL(GAME))(mutation)
+    .then(data => data.launchCycle)
+}
+
+function handleUpdateCycleStateCommand(state, statusMsg, notify, options) {
+  const {
+    lgJWT,
+    lgUser,
+    formatMessage,
+    formatError
+  } = options
+  try {
+    if (!lgJWT || !lgUser || lgUser.roles.indexOf('moderator') < 0) {
+      throw new Error('You are not a moderator.')
+    }
+    notify(formatMessage(statusMsg))
+    return invokeUpdateCycleStateAPI(state, lgJWT)
+      .catch(error => {
+        errorReporter.captureException(error)
+        notify(formatError(`API invocation failed: ${error.message}`))
+      })
+  } catch (errorMessage) {
+    notify(formatError(errorMessage.message))
+  }
+}
+
+export function invoke(argv, notify, options = {}) {
+  const opts = Object.assign({}, defaultInvokeOptions, options)
+  const {
+    formatMessage,
+    formatError,
+    formatUsage
+  } = opts
+  assertFunctions({notify, formatMessage, formatError, formatUsage})
+  let args
+  try {
+    args = parse(argv)
+  } catch (error) {
+    return notify(formatError(error))
```

Comment:
nitpick: unless there some reason we're returning the return value of notify here, maybe the `return` statement should come after `notify`

```
notify(formatError(error))
return
```
Otherwise the return value of this function is a little unpredictable and/or inconsistent.